uber/site_sections/preregistration.py

```diff
@@ -258,6 +258,8 @@ def register_group_member(self, session, group_id, message='', **params):
 
                 badge_being_claimed = group.floating[0]
                 attendee.registered = badge_being_claimed.registered
```

Comment:
To be honest, I'm not sure unassigned Staff/Supporter badges should even _have_ numbers.

Imagine an event where all badge types get pre-assigned a number. Do you really want unassigned badges to have numbers in that case? You'll have just... oodles and oodles of badges no one needs taking up space in your badge range. And that's no good.